src/control/server/mgmt_svc.go

```diff
@@ -899,7 +978,7 @@ func (svc *mgmtSvc) StartRanks(ctx context.Context, req *mgmtpb.RanksReq) (*mgmt
 	case <-time.After(timeout):
 	}
 
-	// either all instances started or Timeout
+	// either all instances st timeout occurred
```

Comment:
nit - typo

Reply:
I will address as misc item in https://github.com/daos-stack/daos/pull/1785